src/PereViader.Scrolller/IStreamingScrolllerService.cs

```diff
@@ -0,0 +1,11 @@
+using PereViader.Scrolller.Responses;
+
+namespace PereViader.Scrolller;
+
+public interface IStreamingScrolllerService
+{
+    IAsyncEnumerable<IEnumerable<SubredditPostResponse>> Subreddit(string subreddit, CancellationToken ctx = default);
+    IAsyncEnumerable<IEnumerable<SubredditResponse>> Discover(bool isNsfw, CancellationToken ctx = default);
+    IAsyncEnumerable<Uri> DiscoverUrls(bool isNsfw, CancellationToken ctx = default);
+    IAsyncEnumerable<Uri> SubredditUrls(string subreddit, CancellationToken ctx = default);
```

Comment:
> Do you have a use case for these ones? If not we could consider removing them from the library to keep it simple.

not really since it's easy enough to retrieve them from the full response. I removed this